content/blog/its-time-to-try-out-pixi/contents.lr

```diff
@@ -0,0 +1,995 @@
+title: It's time to try out pixi!
+---
+author: Eric J. Ma
+---
+body:
+
+I recently switched LlamaBot and my personal website repository to `pixi`,
+and having test-driven it for a few weeks now,
+I think the time is right to do so.
+During this test-drive, I went cold turkey:
+I deleted `~/anaconda` from my home directory,
+leaving me with no choice but to use `pixi` 100%.
+In this post, I'd like to share what I've learned from my perspective
+as a data scientist and software developer.
+
+## What is pixi?
+
+In my mind, pixi is a package management multi-tool.
+`pixi` plays many roles, but here are the big, salient ones:
+
+| Role                | Analogy                                                  |
+| ------------------- | -------------------------------------------------------- |
+| Package installer   | `conda` and `pip`                                        |
```

Comment:
You could possibly also add `pipx` or `apt-get` and `homebrew` which is what we're trying to replace with `pixi global`

Reply:
Yes, I think that's a great point! I have added it as a new line, "global tool installer", which feels categorically different from a package installer alone.